analysis_options.yaml

```diff
@@ -7,14 +7,15 @@
 # See the configuration guide for more
 # https://github.com/dart-lang/sdk/tree/master/pkg/analyzer#configuring-the-analyzer
 #
-# NOTE: Please keep this file in sync with
-# https://github.com/flutter/flutter/blob/master/.analysis_options
+# This file contains the analysis options used by Flutter tools, such as
+# IntelliJ, Android Studio, and the 'flutter analyze' command.
+#
+# This file is derived from the master file in the flutter repo, and should be
+# kept in sync with it.
```

Comment:
we should be able to do a relative import rather than duplicating this

if you can figure out how to make that work, we can then also apply that to the other places where we duplicate this

Reply:
Yeah, I'd love to do that, but I have no idea how to do relative imports here, or how to reference something in the flutter dir (wherever it is).  I've looked at the [docs for the analysis options](https://www.dartlang.org/guides/language/analysis-options), and it doesn't say anything about imports.